src/errorHandler.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Alert, Clipboard } from 'react-native';
+import hathorLib from '@hathor/wallet-lib';
+import * as Sentry from '@sentry/react-native';
+import VersionNumber from 'react-native-version-number';
+
+/**
+ * Send error to Sentry
+ */
+export const sentryReportError = (error) => {
+  Sentry.init({
+    dsn: 'https://c1ebae9159f741e8937abdbfbeba8e8a@o239606.ingest.sentry.io/5304101',
+  });
+  Sentry.withScope((scope) => {
+    scope.setExtra('App version', JSON.stringify(VersionNumber));
+    Sentry.captureException(error);
+  });
+};
+
+/**
+ * Copy access data from storage to the clipboard.
+ * This could be useful to ask the user to copy it and check if
+ * all keys are correctly there.
+ * With this we prevent sending sensitive data to Sentry
+ */
+export const copyData = (error) => {
+  const accessData = JSON.stringify(hathorLib.wallet.getWalletAccessData());
+  Clipboard.setString(`${error.name} - ${error.message}\n${accessData}`);
+};
+
+/**
+ * Capture all JS exceptions that might happen and show an alert to the user
+ */
+export const errorHandler = (error, isFatal) => {
+  if (isFatal) {
+    Alert.alert(
+      'Unexpected error occurred',
```

Comment:
Missing localization markers for all strings.

Reply:
I've added the markers and the translation, however when I do `make compile_msgs` I get errors because the new messages were not translated to danish. 

```
mkdir -p src/locale/pt-br/
npx ttag po2json locale/pt-br/texts.po > src/locale/pt-br/texts.po.json
msgcmp locale/da/texts.po locale/texts.pot
locale/texts.pot:8: this message is used but not defined in locale/da/texts.po
locale/texts.pot:18: this message is used but not defined in locale/da/texts.po
locale/texts.pot:22: this message is used but not defined in locale/da/texts.po
msgcmp: found 3 fatal errors
make: *** [locale/da/texts.po] Error 1
```

@msbrogli Am I missing something here on the translating update process?

Reply:
@pedroferreira1 You need to update the PO files as well. We don't have a `make` command for it; you can just follow the instructions in the README.